src/Plugin.Maui.ScreenRecording/ScreenRecording.android.cs

```diff
@@ -28,6 +28,8 @@ public partial class ScreenRecordingImplementation : MediaProjection.Callback, I
 
 	public bool IsSupported => ProjectionManager is not null;
 
+	public bool IsSavingToGallery { get; private set; }
```

Comment:
I don't think this needs to be public? 
```suggestion
	bool IsSavingToGallery { get; private set; }
```